build/wikia/transformers/elements.mjs

```diff
@@ -13,3 +11,6 @@ module.exports = {
   Gas: 'gas',
   Void: 'void',
 };
+Object.freeze(ELEMENTS);
+
+export default ELEMENTS;
```

Comment:
it's never modified, only used internally. no point doing extra ops every time for something that's already treated in code as a const object with const values...

(same thing on all the other enum mappers)